pkgs/development/libraries/glibc/common.nix

```diff
@@ -37,7 +37,6 @@
 , profilingLibraries ? false
 , withGd ? false
 , withLibcrypt ? false
-, meta
```

Comment:
this is used in locales.nix and info.nix, restore it for now with the default value of {} and i'll think of a nicer glibc expr later